1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,12 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for( let salary in salaries){
+  // I am loving the charecter names from 'FRIENDS'. The best and classic
+  if(salaries[salary] > 50000){
+     console.log({salary: `$${salaries[salary]}`})
```

Comment:
As you are using the string interpolation, you should wrap the whole string you want inside like the suggestion shows. The way you have it now will print objects with the value as a string, this doesn't match the expected output.
```suggestion
     console.log(`${salary}: $${salaries[salary]}`)
```